sql-ballerina/tests/complex-query-test.bal

```diff
@@ -273,6 +275,105 @@ function testDateTime() {
     test:assertEquals(value, expected, "Expected record did not match.");
 }
 
+type ResultDates2 record {
+    time:Date DATE_TYPE;
+    time:TimeOfDay TIME_TYPE;
+    time:Civil TIMESTAMP_TYPE;
+    time:Civil DATETIME_TYPE;
+    time:TimeOfDay TIME_TZ_TYPE;
+    time:Civil TIMESTAMP_TZ_TYPE;
+};
+
+@test:Config {
+    groups: ["query", "query-complex-params"]
+}
+function testDateTime2() {
+    MockClient dbClient = checkpanic new (url = complexQueryDb, user = user, password = password);
+    string insertQuery = string `INSERT INTO DateTimeTypes (row_id, date_type, time_type, timestamp_type,
```

Comment:
Since we are inserting duplicate records for each testDateTime[x] test cases, would it be nice to insert a record in sql script at docker container creation,  rather than insert duplicates in each test cases?

Reply:
Yes. I will update